### PR TITLE
MODORDERS - 192 - API Tests for vendor/access provider status validation

### PR DESCRIPTION
**Pull request by khandramai**
Order creation/updating with inactive/non-existed vendor/access providers testing on folio-testing env with the following steps:
1. All the needed Vendors and Access Providers (APs) created.
create_vendors
2. Successful case for POST PO added (Vendor and APs are existed and active) added to existent "Create Open order with P/E Mix and Electronic lines" test.
3. The following combinations were verified for negative cases both for POST and PUT requests:
- Vendor not found
- Access Provider (AP) not found
- Vendor is inactive
- One of APs is inactive
- Vendor and one of APs are inactive
- Vendor and all the APs are inactive
- Vendor and one of APs are inactive, another one AP not found (this case shown on the screenshots below)
negative_post
negative_put
4. Clean vendors and created order
vendors_delete

### CODE REVIEW

**Comment by craigmcnally** [comment]
I'm getting test failures related to 422 vs 400 response codes:
image

**Comment by khandramai** [comment]
Hi @piotr-kalashuk,
> * Please move vendor body request to pre-request script as template and re-use when creating vendor
I left this as is with reduction vendor request body content.

>     * As there is no difference between vendor and access provider actually, I would propose to create only one active vendor and one inactive and just use these for all tests.
Done. I reduced vendors set to two instances - active and inactive vendor.

>     * Either unset vendor id variables or move them to collection variables as fixed uuids.
Done. I left mentioned ids in environment variables but clean them after test finishing.

File: mod-orders/mod-orders.postman_collection.json [hunk 1]
```diff
@@ -6505,65 +6565,1068 @@
...
+											"raw": "{{protocol}}://{{url}}:{{okapiport}}/orders/composite-orders/{{completeOpenOrderId}}",
+											"protocol": "{{protocol}}",
+											"host": [
+												"{{url}}"
+											],
+											"port": "{{okapiport}}",
+											"path": [
+												"orders",
+												"composite-orders",
+												"{{completeOpenOrderId}}"
```
Comment (piotr-kalashuk) [inline mod-orders/mod-orders.postman_collection.json]: Please use `negativeTestsOpenOrderId` instead of `completeOpenOrderId` for negative tests.

**Comment by craigmcnally** [comment]
Still getting errors, though they have changed:  
image

**Comment by khandramai** [comment]
Hi @craigmcnally,
it seems folio-testing environment wasn't cleaned between tests runs. I see a lot of created and non-deleted orders.
orders

**Comment by khandramai** [comment]
I tried to run tests on my local clean environment right now  - successfully.
tests

**Comment by craigmcnally** [comment]
Is there an easy way to clean the testing env so that I can get these tests to pass?

**Comment by piotr-kalashuk** [comment]
I cleaned up all the orders created in testing and re-run tests collection:
image

**Pull request merged**